include/mbedtls/config_psa.h

```diff
@@ -742,13 +742,11 @@ extern "C" {
 #define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_USE 1
 #define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_IMPORT 1
 #define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_EXPORT 1
-#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE 1
```

Comment:
I'm sorry, but why not? We can derive ECC keys when `ECP_C` is enable, so we should define the `_DERIVE` macro here, shouldn't we? (Same a few lines below of course.)

Reply:
Maybe I misunderstood [this comment](https://github.com/Mbed-TLS/mbedtls/pull/7641#discussion_r1223101505). I can revert this change and check again the original request better

Reply:
`ecp.c` doesn't have a deterministic key generation functionality. The general idea of the MBEDTLS→PSA_WANT translation is to enable similar functionality. `MBEDTLS_ECP_C` has the functionality of `PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_{USE,IMPORT,EXPORT,GENERATE}` but not `DERIVE`.